package.json

```diff
@@ -24,7 +24,8 @@
     "screenshots": "mgon-screenshots --output screenshots",
     "patch": "patch-package",
     "postbuild:studio": "ts-node -s converter/postbuild.ts",
-    "postinstall": "patch-package",
+    "postinstall": "patch-package && npx simple-git-hooks",
+    "prepare": "npx simple-git-hooks",
```

Comment:
from my understanding, it is needed to automatically have Git hooks enabled after install. otherwise, you will need to run it manually after an install (or at least that was my experience). are you seeing a different behavior?

Reply:
But the `prepare` already do this, right?

Reply:
good question. i removed it for now. thank you!